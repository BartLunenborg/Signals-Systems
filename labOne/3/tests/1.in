42 1 1.047198 1 0.523599
