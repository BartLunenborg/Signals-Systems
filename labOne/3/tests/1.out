x(t)=1.93*cos(2*pi*42.00*t+0.79)
